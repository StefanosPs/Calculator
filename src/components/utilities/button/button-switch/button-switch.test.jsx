import React from 'react';
import renderer from 'react-test-renderer';

import ButtonSwitch from './button-switch';
 

it('renders  ButtonSwitch', () => {
    const buttonSwitch = renderer.create(<ButtonSwitch />).toJSON();
    expect(buttonSwitch).toMatchSnapshot();
});