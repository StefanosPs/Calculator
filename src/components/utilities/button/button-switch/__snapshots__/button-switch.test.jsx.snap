// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders  ButtonSwitch 1`] = `
<div
  className="switch float-none"
>
  <h6 />
  <label
    className="switch-label"
  >
    <input
      type="checkbox"
    />
    <span
      className="slider round"
    />
  </label>
</div>
`;
